cv_checker_frontend/src/components/pages/JobDetailsPage.tsx

```
@@ -54,17 +61,73 @@ const JobDetailsPage = ({ job_id }: { job_id: number }) => {
+                <h2 className='font-heading text-xl text-center exsmall:text-2xl sm:text-3xl md:text-4xl font-bold'>{jobDetails?.job_title}</h2>
+              </div>
+              <div className='absolute z-10 bg-white/40 shadow-[inset_23px_-16px_52px_16px_#FFFF] h-[200px] w-full'>
+              </div>
+              <div className='flex flex-col gap-6 absolute z-20 top-[200px] w-full'>
+                <div>
+                  <h3 className='font-heading font-semibold text-center text-2xl'>
+                    Recommended CVs
+                  </h3>
+                  {
+                    jobDetails?.cvs.map((cv) => (
+                      <div className='m-6 w-full'>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)